File: solutions/A1-001.py
import sys

def main():
    # Read all input at once
    input_data = sys.stdin.read().split()
    if len(input_data) < 2:
        return
    
    fname = input_data[0]
    lname = input_data[1]
    
    # Requirement: Hello [Firstname] [Lastname]
    # Requirement: [First 2 chars of Firstname][First 2 chars of Lastname]
    print(f"Hello {fname} {lname}")
    print(f"{fname[:2]}{lname[:2]}")

if __name__ == "__main__":
    main()

File: solutions/A1-001.c
#include <stdio.h>

int main() {
    char fname[100], lname[100];
    scanf("%s %s", fname, lname);
    printf("Hello %s %s\n", fname, lname);
    printf("%.2s%.2s\n", fname, lname);
    return 0;
}